dockerfile-configs/common-components.yaml

```diff
@@ -126,6 +126,14 @@
       kubectl krew install get-all
     info: |
       ketall is to enable `kubectl get-all -n namespace_name`, for listing all resources in some namespace
+  - name:  crictl
+    command: |
+      VERSION="v1.17.0";\
+      curl -fsSLO https://github.com/kubernetes-sigs/cri-tools/releases/download/$VERSION/crictl-$VERSION-linux-amd64.tar.gz;\
```

Comment:
The problem with this installation approach is that crictl is only needed in Gardener to communicate with containerd CRI. However the crictl default configuration points to docker. Crictl in Gardener should point per default to containerd. If containerd is not installed crictl will not work, but that is fine as the user can just use the regular docker commands then. 
Please create the config file for crictl having the configuration for containerd.
Thanks!

Reply:
Hi @danielfoehrKn , thanks for the comments, i've added one config pointing to `unix:///run/containerd/containerd.sock` and copy the config file to crictl default config location in `/etc/crictl.yaml`